d2/src/main/java/com/linkedin/d2/balancer/dualread/DualReadLoadBalancer.java

```diff
@@ -59,16 +61,35 @@ public class DualReadLoadBalancer implements LoadBalancerWithFacilities
   private final LoadBalancerWithFacilities _oldLb;
   private final LoadBalancerWithFacilities _newLb;
   private final DualReadStateManager _dualReadStateManager;
-
+  private ExecutorService _executor;
```

Comment:
nit: newLbExecutor

Reply:
Thanks. Updated

Reply:
Haha I opened an issue about this naming because it used to be `loadBalancerExecutor` and said that it was redundant, since the name of the class is `DualReadLoadBalancer`.